### DISCUSSION TITLE
post/vaultwarden-on-aliyun-serverless/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#31] [Announcements]
# post/vaultwarden-on-aliyun-serverless/

之前试用了一段时间的 Lastpass, 功能上基本都能满足我的需求, 可是免费版电脑手机不能一起使用. 而且刷着 v2ex 就被传教了. 准备自己部署 Bitwarden 来用. 处于稳定性/安全性/价格考虑最后也是选择了部署在云函数上. 没找到网上使用 Serverless 方式部署 Bitwarden 后端的教程(而且有个小坑), 就来自己记录一下.

https://blog.lyc8503.net/post/vaultwarden-on-aliyun-serverless/

**Comment by sxhoio** [comment]
ENABLE_WEBSOCKET=false即可

**Reply by lyc8503** [reply]
确实，新版本提供了这个开关，我在某次升级后已经改用了 `ENABLE_WEBSOCKET=false`，写本文时官方还没提供禁用的方法。
